samples/external-api/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
I'd mention in the README that this extension requires a modern browser. I think up until this PR we were trying to write code so it works in IE11.

Reply:
@jelz I updated PR and made it ES5 only